src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventCounter.cs

```diff
@@ -42,7 +42,7 @@ public EventCounter(string name, EventSource eventSource) : base(name, eventSour
             {
                 bufferedValues[i] = UnusedBufferSlotValue;
             }
```

Comment:
Nit: Array.Fill ?

Reply:
It would need ifdefs because of multitargeting.

Reply:
Ah, ok.